mltsp/predict.py

```diff
@@ -188,46 +188,87 @@ def create_feat_dict_and_list(new_obj, features_to_use, features_extracted):
     return (newFeatures, features_dict)
 
 
-def add_to_predict_results_dict(results_dict, classifier_preds, fname, ts_data,
-                                features_dict, featset_key, n_cols_html_table):
+def add_to_predict_results_dict_classification_proba(
+        results_dict, estimator_preds, fname, ts_data, features_dict,
+        featset_key, n_cols_html_table):
     """
     """
-    # Load model class list
-    all_objs_class_list = list(np.load(
-        os.path.join(cfg.FEATURES_FOLDER, "%s_classes.npy" % featset_key)))
-    sorted_class_list = []
-    for i in sorted(all_objs_class_list):
-        if i not in sorted_class_list:
-            sorted_class_list.append(i)
-    class_probs = classifier_preds[0]
-    class_names = sorted_class_list
-
-    results_str = ("<tr class='pred_results'>"
-                   "<td class='pred_results pred_results_fname_cell'>"
+    # Load model target list
+    all_objs_target_list = list(np.load(
+        os.path.join(cfg.FEATURES_FOLDER, "%s_targets.npy" % featset_key)))
+    sorted_target_list = []
+    for i in sorted(all_objs_target_list):
+        if i not in sorted_target_list:
+            sorted_target_list.append(i)
+    target_probs = estimator_preds[0]
+    target_names = sorted_target_list
+
+    results_str = ("<tr target='pred_results'>"
+                   "<td target='pred_results pred_results_fname_cell'>"
                    "<a href='#'>%s</a></td>") % os.path.basename(fname)
     results_arr = []
 
-    for i in range(len(class_probs)):
+    for i in range(len(target_probs)):
         results_arr.append(
-            [sorted_class_list[i], float(class_probs[i])])
+            [sorted_target_list[i], float(target_probs[i])])
     results_arr.sort(key=itemgetter(1), reverse=True)
 
     for i in range(len(results_arr)):
         if i < n_cols_html_table:
             results_str += """
-                <td class='pred_results'>%s</td>
-                <td class='pred_results'>%s</td>
+                <td target='pred_results'>%s</td>
+                <td target='pred_results'>%s</td>
             """ % (str(results_arr[i][0]), str(results_arr[i][1]))
 
     results_str += "</tr>"
     results_dict[os.path.splitext(os.path.basename(fname))[0]] = {
         "results_str": results_str, "ts_data": ts_data,
-        "features_dict": features_dict, "pred_results_list": results_arr}
+        "features_dict": features_dict, "pred_results": results_arr}
+    return
+
+
+def add_to_predict_results_dict_classification(
+        results_dict, estimator_preds, fname, ts_data, features_dict,
+        featset_key, n_cols_html_table):
+    """
+    """
+    results_str = ("<tr class='pred_results'>"
+                   "<td class='pred_results pred_results_fname_cell'>"
+                   "<a href='#'>%s</a></td>") % os.path.basename(fname)
+    if isinstance(estimator_preds, (list, np.ndarray)):
```

Comment:
Just curious, what are the different types that `estimator_preds` could have here? Ideally it would always be the same (probably a 1d numpy array?).